packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -97,16 +97,35 @@ export class DumbTriggerManager extends Component {
    * OAuth Form success handler. OAuthForm retrieves an account id created by the
    * cozy stack
    * @param  {string}  accountId
+   * @param  {integer} bankId
    */
-  async handleOAuthAccountId(accountId) {
+  async handleOAuthAccountId(accountId, bankId) {
```

Comment:
I am a bit surprised to see the banking notion leak here. I cannot see where this is called though. Would it be possible to have only generic oauth handling in TriggerManager and keep banking specific behaviors in the policy ?

Reply:
Right, and this is not needed. I will keep this bankId in BI policy